decision_maker.go

```diff
@@ -393,36 +397,37 @@ func getValuesFiles(r *release) string {
 		fileList = append(fileList, r.SecretsFiles...)
 	}
 
-	if len(fileList) > 0 {
-		return " -f " + strings.Join(fileList, " -f ")
+	fileListArgs := []string{}
+	for _, file := range fileList {
+		fileListArgs = append(fileListArgs, "-f", file)
 	}
-	return ""
+	return fileListArgs
 }
 
 // getSetValues returns --set params to be used with helm install/upgrade commands
-func getSetValues(r *release) string {
-	result := ""
+func getSetValues(r *release) []string {
+	result := []string{}
 	for k, v := range r.Set {
-		result = result + " --set " + k + "=\"" + strings.Replace(v, ",", "\\,", -1) + "\""
+		result = append(result, "--set", k+"="+strings.Replace(v, ",", "\\,", -1)+"")
 	}
 	return result
 }
 
 // getSetStringValues returns --set-string params to be used with helm install/upgrade commands
-func getSetStringValues(r *release) string {
-	result := ""
+func getSetStringValues(r *release) []string {
+	result := []string{}
 	for k, v := range r.SetString {
-		result = result + " --set-string " + k + "=\"" + strings.Replace(v, ",", "\\,", -1) + "\""
+		result = append(result, "--set-string", k+"="+strings.Replace(v, ",", "\\,", -1)+"")
```

Comment:
Same here.

Reply:
No, it does not break if you enter a space.